aries-site/src/pages/templates/toast-notifications.mdx

```diff
@@ -33,13 +43,31 @@ Everything that you need to know about toasts and more.
 - Requires user action or input
 - Batch updates
```

Comment:
This seems a little fuzzy. Could I not initiate a batch action on 100 items and then see a toast notification when they have all completed? I am thinking of a 1-1 connection between user initiated action and toast notification of completion.